src/cockpit/bridge.py

```diff
@@ -154,6 +154,26 @@ def try_to_receive_stderr():
             os.close(stderr_fd)
 
 
+def setup_logging(debug: bool):
+    """Setup our logger with optional filtering of modules if COCKPIT_DEBUG env is set"""
+
+    cfg = {}
+    modules = set(os.getenv('COCKPIT_DEBUG', '').split(','))
```

Comment:
You should filter out empty names here, as otherwise trailing or doubled commas would show all messages.